src/sensors/scripts/truewind_simple.py

```diff
@@ -20,7 +20,7 @@ class TrueWindNode:
     def __init__(self):
         self._lock = threading.Lock()
         self._odom = rospy.Subscriber("/odometry/filtered", Odometry, self._update_odom)
-        self.wind_direction_sub = rospy.Subscriber("wind_direction", Float32, self._update_wind_direction)
+        self.wind_direction_sub = rospy.Subscriber("relative_wind_direction", Float32, self._update_wind_direction)
```

Comment:
Can you update the simulator to publish on this topic too?

Reply:
FIxed